go/libkb/id_table.go

```diff
@@ -420,14 +414,14 @@ func (l TrackChainLink) IsRemote() bool {
 
 func ParseTrackChainLink(b GenericChainLink) (ret *TrackChainLink, err error) {
 	var tmp string
-	tmp, err = b.payloadJSON.AtPath("body.track.basics.username").GetString()
+	tmp, err = b.GetPayloadJSON().AtPath("body.track.basics.username").GetString()
```

Comment:
This could get called once per function to save some CPU time.

Reply:
Sure!

Reply:
i was going to make the same suggestion....

Reply:
I changed these already.